ci/make_example_size_cmt.py

```diff
@@ -0,0 +1,88 @@
+from typing import Dict, List, Optional, Tuple
+
+import os
+import json
+
+
+header = "| examples | master (KB) | pull request (KB) | diff |"
+sep = "| --- | --- | --- | --- | "
+
+
+def format_size(size: Optional[int]) -> str:
+    if size is None:
+        return "N/A"
+
+    if size == 0:
+        return "0"
+
+    return f"{size / 1024:.3f}"
+
+
+def format_diff_size(
+    master_size: Optional[int], pr_size: Optional[int]
+) -> Tuple[str, bool]:
+    if master_size is None or pr_size is None:
+        return ("N/A", False)
+
+    diff = pr_size - master_size
+
+    if diff == 0:
+        return ("0", False)
+
+    diff_percent = master_size / diff
```

Comment:
```suggestion
    diff_percent = diff / master_size
```
Since "post-size-comment" is run in the context of the master branch, this is untested, but I think the other way would be correct.